presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -242,13 +248,14 @@ public void printFinalInfo()
             }
         }
 
-        // 0:32 [2.12GB, 15M rows] [67MB/s, 463K rows/s]
-        String statsLine = format("%s [%s rows, %s] [%s rows/s, %s]",
-                formatTime(wallTime),
+        // [Measured Latency: client-side: 0:33, server-side: 0:32] [2.12GB, 15M rows] [67MB/s, 463K rows/s]
+        String statsLine = format("[Measured Latency: client-side: %s, server-side: %s] [%s rows, %s] [%s rows/s, %s]",
```

Comment:
I'm worried that the output line become too long. Can we remove `Measured Latency:` or just use `Latency:`?

Reply:
Switching to `Latency:`